src/Api/JsonApi/SingleOrManyDataConverterFactory.cs

```diff
@@ -5,7 +5,7 @@
 namespace Defra.PhaImportNotifications.Api.JsonApi;
 
 /// <summary>
-/// Converts <see cref="SingleOrManyData{T}" /> to/from JSON.
+///     Converts <see cref="SingleOrManyData{T}" /> to/from JSON.
```

Comment:
Nit - this looks like a Rider re-format and not something from csharpier? We'll have to watch such changes

Reply:
It was the `Actions on Save -> Reformat and Cleanup`.